osquery/tables/system/darwin/signature.mm

```diff
@@ -23,6 +22,34 @@
 namespace osquery {
 namespace tables {
 
+// Get the flags to pass to SecStaticCodeCheckValidityWithErrors, depending on
+// the OS version.
+Status getVerifyFlags(SecCSFlags& flags) {
```

Comment:
Also! We're actively working on solving the problems with out use of SQLite. I will make an reference issue very soon that highlights some of what we've learned and plans for solutions.